test/index.test.js

```diff
@@ -1148,7 +1168,8 @@ jobs:
 
             return scm.decorateUrl({
                 scmUri,
-                token: 'mytokenfortesting'
+                token: 'mytokenfortesting',
+                scmContext: 'github.com'
```

Comment:
@kumada626 If we are not changing `decorateUrl` function, what is the reason it needs to take in an additional param? Why does `data-schema` for `decorateUrl` need to be changed?